packages/ts-doctor/workspaces.js

```diff
@@ -17,17 +18,56 @@ exports.builder = _ =>
   _.option('cwd', {
     type: 'path',
     describe: 'The current working directory',
-  });
-
-async function getTsConfig(dir) {
+  })
+    .option('with-build-configs', {
+      type: 'boolean',
+      describe:
+        'Creates a tsconfig.build.json file in each package configured for buildind .d.ts files per package. ' +
+        'Useful for Babel based workflows, where `tsc` is only used to build type defs, not compile source. ' +
+        'This is unfortunately necessary because of how compiler flags interact badly with composite projects, making ' +
+        'it impossible to build _just_ type definitions from the root.',
+    })
+    .option('with-sources-metadata', {
```

Comment:
same here?

Reply:
this is fairly optional i think, not a lot of stuff uses right now.